src/main/java/com/google/maps/PlaceAutocompleteRequest.java

```diff
@@ -67,6 +67,15 @@ public SessionToken(UUID uuid) {
       this.uuid = uuid;
     }
 
+    /**
+     * Construct a session that is a continuation of a previous session.
+     *
+     * @param token The unique token to be used as the identifier for this session.
```

Comment:
I would expect `token` here to be a string representation of a UUID after reading this comment, but that's not the case given the implementation. While this provides convenience, I think this is potentially error-prone. I suggest either clarifying the comment, or keeping the existing API of constructing a `SessionToken` from a `UUID`. The latter seems like a clearer API to me, but if a convenience initializer is useful for others, I'm happy to merge in.

Reply:
would
`@param token The character string to be used as the unique identifier for this session.`
be clear enough?

Reply:
> keeping the existing API
The existing API does not work well if the session identifier is not a UUID.